core/src/test/scala/integration/kafka/admin/TopicCommandIntegrationTest.scala

```diff
@@ -320,6 +320,9 @@ class TopicCommandIntegrationTest extends KafkaServerTestHarness with Logging wi
 
     topicService.alterTopic(new TopicCommandOptions(
       Array("--topic", testTopicName, "--replica-assignment", "5:3,3:1,4:2", "--partitions", "3")))
+    TestUtils.waitUntilTrue(
+      () => brokers.forall(_.metadataCache.getTopicPartitions(testTopicName).size == 3),
+      "Timeout waiting new assignment propagate to broker")
```

Comment:
nit: Timeout waiting [for] new assignment....

Same comment apply to 2 other places in the same test file.

Reply:
Thank you for the suggestions, have rectified all of them.